### ISSUE TITLE
About baseline

### ISSUE THREAD

**Issue by sirosann** [franciszzj/HiLo#4]
How much GPU memory are you training on Hilo baseline?
Thank you.

**Comment by sirosann** [comment]
Thanks for your answer.
I tried to train the HiLo baseline which was loaded by pre-trained mask2former.
However, it did not reach the accuracy of the paper.

Could you release the pre-trained model for the HiLo baseline?

**Comment by franciszzj** [comment]
@sirosann I have updated the pre-trained model of HiLo Baseline in the Readme. If you have any questions, please ask in issue.

**Comment by sirosann** [comment]
Thank you very much for your response. I'm sorry maybe I didn't make myself clear.

I wanted to reproduce this result through local training. So I was hoping you could provide a pre-trained model of mask2former.

By the way, I think this [code](https://github.com/franciszzj/HiLo/blob/main/tools/change_model.py) doesn't seem to apply the pre-trained mask2former model to HiLo baseline training.

**Comment by franciszzj** [comment]
You can find the converted mask2former models that I have prepared in README.
These models were converted using `change_model.py`, which you can further explore to understand how the conversion was done. Alternatively, you can use the converted models I provided to start training directly.
Note, all the pre-trained mask2former models are sourced from mmdetection: https://github.com/open-mmlab/mmdetection/tree/main/configs/mask2former#panoptic-segmentation

**Comment by sirosann** [comment]
Thanks for your reply. It helps a lot.
